lib/use-cases/GetDropbox.test.js

```diff
@@ -69,6 +69,19 @@ describe('use-case/get dropbox', () => {
         const dropbox = await getDropbox('abc123456');
         expect(dropbox.hasUploads).toBe(true);
       });
+
+      it('filters empty documents', async () => {
+        options.gateways.evidenceStore.getByDropboxId.mockImplementation(() => {
+          return Promise.resolve([
+            new Document({
+              id: 'def123456'
+            })
+          ]);
+        });
+        const dropbox = await getDropbox('abc123456');
+        console.log(dropbox);
```

Comment:
Not sure if this was left in accidentally?

Reply:
Ha yep, thanks